arguebuf/models/node.py

```diff
@@ -557,6 +581,30 @@ def __repr__(self):
             ],
         )
 
+    @classmethod
+    def from_sadface(
+            cls,
+            obj: sadface.Node,
+            nlp: t.Optional[t.Callable[[str], t.Any]] = None,
+    ) -> SchemeNode:
+        """Generate SchemeNode object from SADFace Node object."""
+        name = None
+        if obj["name"] == "Support":
+            name = Support
+        elif obj["name"] == "Attack":
+            name = Attack
+        elif obj["name"] == "Rephrase":
+            name = Rephrase
+        elif obj["name"] == "Preference":
+            name = Preference
+
+        return cls(
+            id=obj["id"],
+            userdata=obj["metadata"],
+            metadata=Metadata(pendulum.now(), pendulum.now()),
```

Comment:
I would use the same approach as for atom nodes: Create the timestamp first (`timestamp = (pendulum.now())`) and then assign it to the metadata object. Otherwise, the two times will be slightly off.